### ISSUE TITLE
Termistor type 12 missing

### ISSUE THREAD

**Issue by mrvn** [MarlinFirmware/Marlin#2758]
Setting 
    #define TEMP_SENSOR_0 12
gives:

thermistortables.h:1094:22: error: 'temptable_12' was not declared in this scope
 #define _TT_NAME(_N) temptable_ ## _N

**Comment by AnHardt** [comment]
Yep.
12 is for unknown reasons only defined for the bed.
In thermistortable.h search for `temptable_12`. 
Two lines above you will find

```
#if (THERMISTORBED == 12)
```

replace it with

```
#if (THERMISTORHEATER_0 == 12) || (THERMISTORHEATER_1 == 12) || (THERMISTORHEATER_2 == 12) || (THERMISTORHEATER_3 == 12) || (THERMISTORBED == 12)
```

PR will follow tomorrow.

**Comment by AnHardt** [comment]
Fixed by PR #2760
